packages/fiori/cypress/specs/NotificationList.cy.tsx

```diff
@@ -152,3 +152,26 @@ describe("Keyboard Navigation", () => {
 		cy.get("#item11").should("be.focused");
 	});
 });
+
+describe("Events", () => {
+	beforeEach(() => {
+		cy.mount(<Sample />);
+	});
+
+	it("Test click on NavigationListItem", () => {
+
+		cy.get("#item11")
+			.as("notificationLIstItem");
+
+		cy.get("@notificationLIstItem")
+			.then(nli => {
+				nli.get(0).addEventListener("click", cy.stub().as("clicked"));
```

Comment:
The mouse click will be fired as of now, without this change.
What is new is the keydown, keyup thing, that won't work as of now and better to be also tested.